cypress/e2e/brewery/StopScenarioRun.cy.js

```diff
@@ -34,4 +46,18 @@ describe('can cancel simulation run', () => {
     ScenarioParameters.getLaunchButton().should('be.visible');
     Scenarios.getDashboardPlaceholder().should('have.text', FAILED_SCENARIO_RUN);
   });
+
+  it('create scenario, launch it and delete it during it is running', () => {
+    ScenarioParameters.expandParametersAccordion();
+    Scenarios.createScenario(secondScenarioName, true, DATASET.BREWERY_ADT, RUN_TEMPLATE.BREWERY_PARAMETERS).then(
+      (value) => {
+        const scenarioId = value.scenarioCreatedId;
+        ScenarioParameters.launch({ runOptions, saveAndLaunch: true });
+        ScenarioManager.switchToScenarioManager();
+        ScenarioManager.getScenarioAccordion(scenarioId).click();
+        ScenarioManager.getScenarioRunStatus(scenarioId, SCENARIO_STATUS.RUNNING);
+      }
+    );
+    ScenarioManager.deleteScenario(secondScenarioName, true);
```

Comment:
Is this line outside of the `createScenario` block on purpose?

Reply:
In other sens, there is no interest to let it inside the block, so I let it outside.